installed-packages.json

```diff
@@ -7,7 +7,7 @@
         },
```

Comment:
Out of scope for this PR, but...

What would you think about adjusting the generation of this "installed-packages.json" file to only list the dependencies that we explicitly include in our composer.json? We initially went with including the versions of all installed packages, but I think the signal-to-noise ratio is too poor to be worth it... I wonder if just seeing the version changes of what we explicitly define as our direct dependencies would be better.

Reply:
Sounds good to me.